nsv13/code/game/machinery/iff_console.dm

```diff
@@ -33,6 +33,8 @@ If someone hacks it, you can always rebuild it.
 /obj/machinery/computer/iff_console/LateInitialize()
 	. = ..()
 	var/obj/structure/overmap/OM = get_overmap()
+	if(OM.faction == "syndicate") //In case the console gets destroyed and needs to be rebuilt
+		start_emagged = TRUE
```

Comment:
```suggestion
	if(OM.faction == "syndicate") //In case the console gets destroyed and needs to be rebuilt
		start_emagged = TRUE
		radio_channel = RADIO_CHANNEL_SYNDICATE
```
Should include the comms as well

Reply:
> Should include the comms as well



> This should go by if it is a ship that is hammerlocked instead, as the current implementation will have issues with non-syndicate boarding targets, ships with flipped IFFs and PvP syndicate vessels.

Suggested changes applied.